scripts/docker/ui-components.ts

```
@@ -1,24 +1,137 @@
+
+    return result.join('') + COLORS.RESET;
+}
 
 /**
- * Prints the Docker whale ASCII art header
+ * Interpolates between two RGB ANSI color codes
+ */
+function interpolateColors(color1: string, color2: string, t: number): string {
+    // Extract RGB values from ANSI codes like \x1b[38;2;250;162;193m
+    const extractRGB = (color: string) => {
+        const match = color.match(/\x1b\[38;2;(\d+);(\d+);(\d+)m/);
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)